fetch.bs

```diff
@@ -5517,10 +5560,13 @@ run these steps:
   <a for="fetch params">canceled</a>:
 
   <ol>
-   <li><p>If <var>request</var>'s <a for=request>window</a> is "<code>no-window</code>" and
-   <var>request</var>'s <a for=request>redirect mode</a> is "<code>error</code>", then set
-   <var>httpFetchParams</var> to <var>fetchParams</var> and <var>httpRequest</var> to
-   <var>request</var>.
+   <li>
+    <p>If <var>request</var>'s <a for=request>traversable for user prompts</a> is "<code>no-traversable</code>" and
+    <var>request</var>'s <a for=request>redirect mode</a> is "<code>error</code>", then set
+    <var>httpFetchParams</var> to <var>fetchParams</var> and <var>httpRequest</var> to
+    <var>request</var>.
+
+    <p class=note>If user prompts are possible, then we need to clone <var>request</var> because ???
```

Comment:
Because it's one of the scenarios where we might have to repeat request.

Reply:
I added that, although I'm not 100% sure I understand when that would happen. Who gets the request the first time, and who gets it the second time? Does this occur for all 3 possible prompt causes (WWW-Authenticate, PACs, and TLS client certs) or just some of them?

Reply:
It happens when you do a request and the server returns a challenge. Then you have to repeat the request addressing the challenge. This happens with HTTP authentication at least. I'm not sure about client certificates as they happen at the TLS level, so maybe not?